portfolio/src/main/java/com/google/sps/data/CommentList.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+
```

Comment:
Be sure to include Javadoc describing this class!

Reply:
Done!